docs/source/configuring.md

```diff
@@ -272,6 +272,13 @@ In some configurations (often on internal networks) users may need Rover to comm
 - The `--insecure-accept-invalid-hostnames` flag will disable hostname validation. If hostname verification is not used, any valid certificate for any site will be trusted for use from any other. This introduces a significant vulnerability to man-in-the-middle attacks.
 
 - The `--insecure-accept-invalid-certs` flag will disable certificate validation. If invalid certificates are trusted, any certificate for any site will be trusted for use. This includes expired certificates. This introduces significant vulnerabilities, and should only be used as a last resort.
+
+## Increase the client timeout
+
+Sometimes, you may want to make queries for large amounts of data that may take some extra time for the Studio API to process.
```

Comment:
Recommendation (feel free to adjust anything I've gotten a little wrong or you disagree with):

## Increasing command timeout

By default, Rover times out requests to the Apollo Studio API after 30 seconds. If you're executing a command that might take longer than 30 seconds to process, you can increase this timeout with the `--client-timeout` option:

```bash
rover lengthy command (whatever that would be) --client-timeout=60 # in seconds
```